tools/taskcluster/schedule-master-build.py

```diff
@@ -51,7 +51,44 @@ def generate_ui_test_task(dependencies):
 		description = "Run UI tests for Focus/Klar for Android.",
 		command = ('echo "--" > .adjust_token'
 			' && ./gradlew --no-daemon clean assembleFocusWebviewUniversalDebug assembleFocusWebviewUniversalDebugAndroidTest'
-			' && tools/taskcluster/execute-firebase-test.sh'),
+			' && ./tools/taskcluster/google-firebase-testlab-login.sh'
+			' && tools/taskcluster/execute-firebase-test.sh focusWebviewUniversal app-focus-webview-universal-debug model=sailfish,version=26 model=Nexus5X,version=23 model=Nexus9,version=25 model=sailfish,version=25'),
+		dependencies = dependencies,
+		scopes = [ 'secrets:get:project/focus/firebase' ],
+		artifacts = {
+			"public": {
+				"type": "directory",
+				"path": "/opt/focus-android/test_artifacts",
+				"expires": taskcluster.stringDate(taskcluster.fromNow('1 week'))
+			}
+		})
+
+def generate_gecko_X86_ui_test_task(dependencies):
+	return taskcluster.slugId(), generate_task(
+		name = "(Focus for Android) UI tests",
+		description = "Run UI tests for Klar Gecko X86 for Android.",
+		command = ('echo "--" > .adjust_token'
+			' && ./gradlew --no-daemon clean assembleklarGeckoX86Debug assembleklarGeckoX86DebugAndroidTest'
+			' && ./tools/taskcluster/google-firebase-testlab-login.sh'
+			' && tools/taskcluster/execute-firebase-test.sh klarGeckoX86 app-klar-gecko-x86-debug model=Nexus5X,version=23'),
```

Comment:
I'm confused. Isn't the Nexus 5X an ARM device? Or is this running an emulator?

Reply:
Yes, actually 5X is an emulator only in Google Firebase.
![screen shot 2018-03-26 at 3 52 24 pm](https://user-images.githubusercontent.com/6930774/37929340-bcd79ea4-310d-11e8-9f1d-f4c9c85dcbde.png)

Reply:
I wish they would give it more obvious names..